### PR TITLE
PBS Version In Response Header

### PR DESCRIPTION
**Pull request by SyntaxNode**
Implements https://github.com/prebid/prebid-server/issues/2010

### CODE REVIEW

File: endpoints/openrtb2/amp_auction_test.go
Comment (SyntaxNode) [inline endpoints/openrtb2/amp_auction_test.go:1337]: There was no need to export this method.

File: endpoints/openrtb2/video_auction_test.go
Comment (SyntaxNode) [inline endpoints/openrtb2/video_auction_test.go:31]: Added a test loading helper method like the auction and amp endpoint have.

File: version/xprebidheader.go
Comment (SyntaxNode) [inline version/xprebidheader.go:53]: Copied/pasted code from another file. No change to BuildXPrebidHeaderForRequest or writeNameVersionRecord.

File: endpoints/openrtb2/amp_auction_test.go
Comment (AlexBVolcy) [inline endpoints/openrtb2/amp_auction_test.go:1368]: For my own understanding, what is happening here with `func(http.Header)`? I haven't seen this syntax with `func(something)` and see it in a couple of places here in this PR.

**Review by AlexBVolcy — COMMENTED** [review]
This looks good to me! I checked code coverage against the new additions you made and that looks good too. I'm going to wait until Gus reviews to see if I may have missed something.

**Review by guscarreon — COMMENTED** [review]
Looks pretty good. I'm ready to approve after merge conflicts get resolved.

File: endpoints/openrtb2/amp_auction_test.go
Comment (SyntaxNode) [reply endpoints/openrtb2/amp_auction_test.go:1372]: Good question. I am defining a function pointer variable. This is the same format as a normal function header, but I'm omitting the variable name since it doesn't matter in this context. In the test cases, I'm defining anonymous methods and need to provide a variable name for use in the body.

I'm doing this here so that I can use the `.Set()` method for adding headers to the map. Since it is a really just a map, I could get away with a map literal.. but there is something formatting performed by the headers class and I want to be sure I'm using the exact same code paths for the test. I could also use a map and then loop through in making calls to `.Set()`, but I thought this was easier and more direct. It's an implementation choice and I wouldn't comment on anyone using an alternate approach. It's just that this is what looks good / makes sense to me.

**Review by bsardo — APPROVED** [review]
LGTM

**Review by guscarreon — APPROVED** [review]
LGTM

**Pull request merged**
